### PR TITLE
Add a new integration document for duwi

### PR DESCRIPTION
**Pull request by duwi2024**
## Proposed change

Following the insightful feedback provided by the Home Assistant Team on my initial submission, I have made several enhancements to the Duwi Integration PR for reevaluation. These changes aim to align with Home Assistant's standards and development procedures more closely. Notably, I've adhered strictly to the PR template, ensuring every required detail is meticulously reported for unequivocal understanding and assessment.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
- [x] I've opened up a PR to add logos and icons in the Brands repository.
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/118012
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #https://github.com/home-assistant/brands/pull/5491

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the [Home Assistant documentation standards][standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/duwi.markdown [hunk 1]
```diff
@@ -0,0 +1,61 @@
+---
+title: Duwi
+description: The Duwi Integration allows users to easily connect and manage their Duwi Smart devices directly from Home Assistant. Control, monitor, and automate your Duwi Smart-enabled devices like lights, switches, and covers. Simplify your home automation tasks with the Duwi Integration.
+ha_category:
+- Cover
+- Light
+- Switch
+ha_iot_class: Cloud Push
+ha_release: 2024.5.4
```
Comment (c0ffeeca7) [inline source/_integrations/duwi.markdown]:
```suggestion
ha_release: 2024.5.4
```
Why is the release in the past?

**Comment by c0ffeeca7** [comment]
Please remove the duwi icon from the 'green' folder. It does not belong there.

**Comment by Ledgerbiggg** [comment]
@c0ffeeca7 I've made the requested changes. Could you please take another look when you have a moment? Thank you!

**Comment by frenck** [comment]
@duwi2024 Why did you close this PR? There is still a parent PR linked & open.

**Comment by Ledgerbiggg** [comment]
Sorry for closing the PR earlier. I will continue to submit my PR.😢

**Comment by c0ffeeca7** [comment]
> @c0ffeeca7 I've made the requested changes. Could you please take another look when you have a moment? Thank you!

I often wait with starting the docs review until the code review has been completed. It is more efficient. During code review, often changes require changes to the docs.

**Comment by Ledgerbiggg** [comment]
Okay, I understand. I am deeply sorry for my behavior and any inconvenience caused to you. I will continue to submit the PR for the document warehouse. I am truly sorry

**Pull request closed without merging**